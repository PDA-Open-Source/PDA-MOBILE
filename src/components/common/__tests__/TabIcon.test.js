import React from 'react';
import renderer from 'react-test-renderer';
import TabIcon from '../TabIcon';
jest.mock('react-native-device-info');
describe('<TabIcon/>',()=>{
  it('TabIcon should render without crash',()=>{
    let snap = renderer.create(<TabIcon/>);
    expect(snap.toJSON()).toMatchSnapshot();
  });
});