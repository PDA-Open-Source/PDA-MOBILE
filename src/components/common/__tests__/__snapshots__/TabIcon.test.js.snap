// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabIcon/> TabIcon should render without crash 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexDirection": "column",
        "justifyContent": "center",
      },
      Object {
        "alignItems": "center",
        "backgroundColor": "#ffffff",
        "height": "100%",
        "justifyContent": "center",
        "paddingTop": 6,
        "width": "100%",
      },
    ]
  }
>
  <Image
    resizeMode="contain"
    source={0}
    style={
      Object {
        "height": 30,
      }
    }
  />
  <Text
    style={
      Array [
        Object {
          "fontFamily": "Roboto-Regular",
          "fontSize": 15.714285714285714,
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {
          "color": "#CCCCCC",
          "flex": 1,
          "paddingVertical": 2,
        },
      ]
    }
  />
</View>
`;
